dist/tools/pyterm/pyterm

```diff
@@ -126,7 +128,7 @@ class SerCmd(cmd.Cmd):
         self.json_regs = dict()
         self.init_cmd = []
         self.load_config()
-        if self.fmt_str == None:
+        if not hasattr(self, "fmt_str") or self.fmt_str == None:
```

Comment:
With `==`, you could have different behavior if `fmt_str` or `formatter` are instances of classes with special implementation of  the comparison operator `__eq__`. With `is`, you really check if they are set or not. It's also more readable.